satellite/gracefulexit/chore_test.go

```diff
@@ -77,9 +91,32 @@ func TestChore(t *testing.T) {
 			require.Len(t, incompleteTransfers, 0)
 		}
 
-		nodeIDs, err = satellite.Overlay.DB.GetExitingNodesLoopIncomplete(ctx)
+		exitingNodes, err = satellite.Overlay.DB.GetExitingNodes(ctx)
 		require.NoError(t, err)
+		nodeIDs = make(storj.NodeIDList, 0, len(exitingNodes))
+		for _, exitingNode := range exitingNodes {
+			if exitingNode.ExitLoopCompletedAt == nil {
+				nodeIDs = append(nodeIDs, exitingNode.NodeID)
+			}
+		}
 		require.Len(t, nodeIDs, 0)
+
+		satellite.GracefulExit.Chore.Loop.Pause()
+		err = satellite.DB.GracefulExit().IncrementProgress(ctx, exitingNode.ID(), 0, 0, 0)
+		require.NoError(t, err)
+
+		// node should fail graceful exit if it has been inactive for maximum inactive time frame since last activity
+		time.Sleep(maximumInActiveTimeFrame)
```

Comment:
nit - we probably don't need to worry about this, but maybe we should sleep for slightly longer than `maximumInactiveTimeFrame`
Also nit - `InActive` -> `Inactive`